proxstar/__init__.py

```diff
@@ -194,7 +195,6 @@ def vm_details(vmid):
 @auth.oidc_auth
 def vm_power(vmid, action):
     user = User(session['userinfo']['preferred_username'])
-    proxmox = connect_proxmox()
```

Comment:
Um we probably want to run `connect_proxmox()`. The variable result `proxmox` is probably unnecessary

Reply:
All three re-added without assignment